client/homebrew/pages/vaultPage/vaultPage.jsx

```diff
@@ -0,0 +1,478 @@
+require('./vaultPage.less');
+
+const React = require('react');
+const { useState, useEffect, useRef } = React;
+const cx = require('classnames');
```

Comment:
Shouldn't need to use `cx`. This was for very old React.

Reply:
replaced with cleaner use of classes